WTWSMS/common/minor_titles/00_minor_titles.txt

```diff
@@ -691,11 +693,6 @@ title_despot = {
 			has_landed_title = e_byzantium
 			has_landed_title = e_roman_empire
 		}
-		OR = {
-			religion_group = christian
-			religion_group = jewish_group
-			religion_group = pagan_group
-		}
```

Comment:
You forget that only certain religions can restore the roman titles. The whole mod's roman feature is based around some cultures and religious restrictions.